### PR TITLE
Refactor/eagle eye checks

### PR DESCRIPTION
**Pull request by irmakerdem**
What does this PR do?
- Fixes spacing and syntax
- Adjusts description of 2 it blocks in Sleep-test.js
- Removes unnecessary variables

Where should your reviewer start?
- Line 32 in index.html
- Line 2 in Activity.js
- Line 11 in Hydration.js
- Line 11 in Sleep.js
- Line 7 in UserRepository.js
- Line 1 in apiCalls.js
- Line 112 in styles.css
- Line 51 in scripts.js
- Line 1 in Sleep-data.js
- Line 6 in Activity-test.js
- Line 3 in Hydration-test.js
- Line 194 in Sleep-test.js

What (if any) features are you implementing?
- N/A

What (if anything) did you refactor?
- Final run through of all files for syntax consistencies and removing unused variables

Were there any issues that arose?
- N/A

Is there anything that you need from your teammate?
- N/A

Any other comments, questions, or concerns?
- N/A

### CODE REVIEW

File: test-data/Sleep-data.js [hunk 1]
```diff
@@ -1,113 +1,112 @@
...
-      "sleepQuality": 5.5
-    },
-    {
-      "userID": 2,
-      "date": "2019/06/23",
-      "hoursSlept": 8.0,
-      "sleepQuality": 1.6
-    },
-  ];
-
```
Comment (catlynbowles) [inline test-data/Sleep-data.js:111]: ooooh the eagle has left the nest people, caw-caw! (bird sound) 🦅

File: test/Activity-test.js [hunk 2]
```diff
@@ -122,72 +123,72 @@ describe('Activity', () => {
...
-      "flightsOfStairs": 15
-    },
-    {
-      "userID": 1,
-      "date": "2022/01/22",
-      "numSteps": 0,
-      "minutesActive": 96,
-      "flightsOfStairs": 31
-    }]);
+    expect(activity.getActivityDataById(1)).to.deep.equal([
```
Comment (catlynbowles) [inline test/Activity-test.js:128]: wow thanks for fixing all these tabbing issues wouldn't have caught this. did you know: Eagles can see clearly about eight times as far as humans can, allowing them to spot and focus in on a rabbit or other animal at a distance of about two miles. 👀

**Review by catlynbowles — APPROVED** [review]

File: test/Sleep-test.js [hunk 3]
```diff
@@ -196,12 +191,12 @@ describe('Sleep', () => {
...
   });
 
-  it('should be able to test whether the date is valid and also return a message if it not', () => {
+  it('should be able to test whether the date is valid for hours slept and also return a message if it not', () => {
     expect(sleep.acquireWeeklyHoursSlept(sleepData1, "2022/06/04")).to.be.a('string');
     expect(sleep.acquireWeeklyHoursSlept(sleepData1, "2022/06/04")).to.equal('Invalid date!')
   });
 
-  it('should be able to test whether the date is valid and also return a message if it not', () => {
+  it('should be able to test whether the date is valid for sleep quality and also return a message if it not', () => {
```
Comment (ASands17) [inline test/Sleep-test.js:199]: I love that you got specific with these descriptors! Good catch! 🤹🏼‍♀️

File: dist/index.html [hunk 4]
```diff
@@ -30,37 +29,37 @@ <h2 role='presentation'class="welcome-text" id="welcomeText">Welcome, name!</h2>
...
-              <label class="sleep-quality" id="sleepQuality">Add Sleep Quality:</label><br />
-              <input class="sleep-quality-input" id="sleepQualityInput" type="text" placeholder="Sleep Quality" /><br /><br />
-              <input class="submit-sleep" id="submitSleep" type="submit" value="Submit" />
+              <label class="sleep-date" id="sleepDate">Add Date:</label><br/>
+              <input class="sleep-date-input" id="sleepDateInput" type="text" placeholder="dd/mm/yyyy"/><br/>
+              <label class="sleep-hours" id="sleepHours">Add Sleep Hours:</label><br/>
+              <input class="sleep-hours-input" id="sleepHoursInput" type="text" placeholder="Number of Hours"/><br/>
+              <label class="sleep-quality" id="sleepQuality">Add Sleep Quality:</label><br/>
+              <input class="sleep-quality-input" id="sleepQualityInput" type="text" placeholder="Sleep Quality"/><br/><br/>
+              <input class="submit-sleep" id="submitSleep" type="submit" value="Submit"/>
```
Comment (ASands17) [inline dist/index.html:38]: Thanks for cleaning up this formatting! It looks a lot more clean now. 🧼

**Pull request merged**